service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/PlacementController.java

```diff
@@ -180,7 +213,13 @@ private boolean isUpdatingExistingPlacement(Placement previousPlacement, Placeme
             && newPlacement.getApplication().equals(previousPlacement.getApplication());
     }
 
-    private List<String> getBinaryUrlsForNoticeOfPlacementOrder(List<Element<Placement>> placements) {
-        return placementService.getBinaryUrlsForOrderAndNotices(placements, NOTICE_OF_PLACEMENT_ORDER);
+    private List<String> getUpdatedDocumentsUrls(CaseData caseData, CaseData caseDataBefore,
```

Comment:
ideally this one could go to placement service, and getBinaryUrlsForOrderAndNotices could be private method in service

Reply:
done